src/pages/test-page.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react"
+import { Layout } from "../components/Layout/Layout"
+import SettingsModal from "../components/SettingsModal/SettingsModal"
+
+const TestPage = () => {
+  const [modalActive, setModalActive] = useState<boolean>(false)
```

Comment:
I have a small question on this.. I am currently struggling with moving sections of code that rely on the local 'usestate' into their own files/custom functions. The method used in this component however, relies on us creating two usestate hooks (one within the custom hook and one within the component), to manage the same thing.. therefore I'm not certain this is efficient. @christoff-linde @brian-david if you have some time later this week, I'd love to talk about the best way to handle this ^^